internal/prober/multihttp/script.go

```diff
@@ -17,26 +16,31 @@ import (
 var templateFS embed.FS
 
 func buildUrl(req *sm.MultiHttpEntryRequest) string {
-	// If we are here, the request has already been validated, and the URL
-	// should be valid. This function should never return an error.
-	u, _ := url.Parse(req.Url)
-
 	var buf strings.Builder
 
 	for _, field := range req.QueryFields {
 		if buf.Len() > 0 {
 			buf.WriteByte('&')
 		}
-		buf.WriteString(url.QueryEscape(field.Name))
```

Comment:
I've been mulling this since yesterday, and I think query escaping (if it needs to happen), has to happen at runtime in the JS. I'm thinking this because

- If a variable is present in the URL, we don't want to escape the characters the signify a variable (`${}`)
- If a variable is present in the URL and resolves to a value that needs to be escaped, we'll need to do that after the variable has a value. So the output JS would need to be something like

```
const foo = "#(%#*)"
const url = "https://www.example.com/${foo}"

http.post(escapeString(url))
```

Instead of doing it ahead of time when building the script